hieradata_aws/integration.yaml

```diff
@@ -91,6 +91,10 @@ govuk::apps::whitehall::basic_auth_credentials: "%{hiera('http_username')}:%{hie
 govuk::apps::whitehall::highlight_words_to_avoid: true
 govuk::apps::whitehall::govuk_notify_template_id: "759acac6-da53-4a19-b591-b7538c7c39de"
 govuk::apps::whitehall::aws_s3_bucket_name: 'govuk-integration-whitehall-csvs'
+# DB hostname for whitehall backend
+govuk::apps::whitehall::admin_db_hostname: 'whitehall-mysql'
+# DB hostname for whitehall frontend
+govuk::apps::whitehall::db_hostname: 'whitehall-mysql'
```

Comment:
As this is MySQL, we'll also need to provide the passwords for each DB, like in https://github.com/alphagov/govuk-puppet/pull/11407/files#diff-32004bbd56816fd524912aacd8994cc4ebcc03172b2fec591f92134b60b9b152R47. 👀 

Apologies if you're already on the case!

Reply:
Hmm I've added in the passwords and it's made whitehall, quite unhappy!

I don't suppose you know if I've named something wrong that would be causing those problems?

Reply:
Ah, I approved too soon then 😁 no, the changes here look good to me. What errors were you seeing?